lib/turn.rb

```diff
@@ -7,18 +7,22 @@ def compare(guess)
     colors_correct = 0
     colors_in_correct_positions = 0
 
-    guess = 'rrgb'
-    array = guess.split ''
-    p array
+    hidden_code = ['g', 'r', 'r', 'b']
+    guess = 'rrgb' # This will probably wind up being equal to guess.gets from the game class
+    individual_guesses = guess.split ''
 
+    individual_guesses.map.with_index {|color, i| color == hidden_code[i]}
 
 
+    colors_correct += (individual_guesses & hidden_code.flatten).count
```

Comment:
Doesn't hurt to have it here but I think the .flatten could be taken out. Nice use of the '&' operator.